src/lsp/superbol_free_lib/project.ml

```diff
@@ -145,7 +145,9 @@ let contributes =
 
             Manifest.PROPERTY.string "superbol.path"
               ~default:"superbol-free"
-              ~description: "Path to the `superbol` command"
+              ~description:
+              "Name of the `superbol` command; path can be relative (deprecated) or \
+               absolute."
```

Comment:
This does not seem to convey the idea that one can put an executable name that will be search in `PATH`. Is it actually allowed?

Reply:
Yes, it is allowed

Reply:
Then I'd suggest something like "Name of the `superbol` executable if available in PATH; otherwise, may be an absolute path."